sales/sales.js

```diff
@@ -1,99 +1,83 @@
-var express = require('express');
-var http = require('http');
-var os = require('os');
+'use strict';
 
-var salesData = require('./data');
-var consul = require('./consul');
+const Consulite = require('consulite');
+const Express = require('express');
+const Customers = require('./lib/customers');
+const Data = require('./lib/data');
 
-var app = express();
-var customersHosts = [];
+const app = Express();
```

Comment:
Makes sense. We do want to demonstrate that mongodb works well with ContainerPilot... we were thinking of adding a service in the workshop, but maybe we should do a separate standalone service.